exercises/practice/pangram/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "authors": [
+    "BNAndras"
+  ],
+  "files": {
+    "solution": [
+      "src/lib.cairo"
+    ],
+    "test": [
+      "tests/pangram.cairo"
+    ],
+    "example": [
+      ".meta/example.cairo"
+    ]
```

Comment:
Missing invalidator

Reply:
Strange that configlet create doesn't add the invalidator file.

Reply:
I agree, it's probably a missing feature in the script